src/mindlessgen/qm/orca.py

```diff
@@ -28,6 +31,10 @@ def __init__(self, path: str | Path, orcacfg: ORCAConfig) -> None:
         else:
             raise TypeError("orca_path should be a string or a Path object.")
         self.cfg = orcacfg
+        self.xtb_cfg = xtb_config
+        self.xtb_driver_enabled = bool(xtb_config) and bool(
+            getattr(self.cfg, "use_xtb_driver", False)
+        )
```

Comment:
Why not use directly this attribute of the class? It's always set anyway?

```suggestion
        self.cfg.use_xtb_driver
```

Reply:
What's the benefit of using `bool(xtb_config)`?

Reply:
I'd even say that it's unnecessary to set `self.xtb_driver_enabled` as an own config entry there.